src/jobs/generation/MetaGenerator.groovy

```diff
@@ -394,7 +394,7 @@ repos.each { repoInfo ->
                     // If not a PR, then the branch name should be the target branch
                     if (isPRTest) {
                         if (isVSTS) {
-                            // TODO: VSTS PR branch
+                            branch(vstsBranch)
```

Comment:
This looks incorrect.  This is a variable here that will be expanded when this script runs, rather than when the generated job runs.  You want '${vstsBranch}' probably